chameleon/src/main/java/org/hibernate/omm/ast/MongoSelectQueryAstTranslator.java

```diff
@@ -152,7 +152,7 @@ public void visitQuerySpec(QuerySpec querySpec) {
             //if ( queryPartForRowNumbering == null ) {
             //visitForUpdateClause( querySpec );
             //}
-            appendSql(" } ], cursor: {} }");
```

Comment:
No longer needed

Reply:
since which version? Removing it will end up with testing failure (`org.hibernate.omm.join.SimpleJoinTests`) for the docker image is still `mongo:5.0.28`. If our library aims to support all versions (minimum version is v3.0 as per `org.hibernate.omm.dialect.MongoDialect`), we need to render JSON based on the current Mongo version. It is exposed by 'dialect' variable in the parent class.